README.md

```diff
@@ -8,4 +8,35 @@ The fastest way to share your notebook with someone.
 - Supports uploading and rendering both Jupyter Notebook and
   RMarkdown files.
 - Opt-in annotation support via [hypothes.is](https://hypothes.is/).
-- Opt-in to allow discovery of your notebooks for search engines.
\ No newline at end of file
+- Opt-in to allow discovery of your notebooks for search engines.
+
+
+There are two ways to share your Notebooks
+
+1. You can upload your notebook easily via the web interface at [notebooksharing.space](https://notebooksharing.space/) *(No Sign up required)*
+2. `nbss-upload` Command-Line Tool
+
+
+## CLI
+
+`nbss-upload` is available on [PyPI](https://pypi.org/project/nbss-upload/), and can be installed with pip.
```

Comment:
Link to GitHub as well, at github.com/notebook-sharing-space/nbss-upload?

Reply:
My bad I didn't see that. Should I just link it to the second point where command line is mentioned or Do you have any suggestions?

Reply:
Yeah, just linking it on the second point seems good :)